src/libraries/System.Net.Http/src/Resources/Strings.resx

```diff
@@ -264,6 +264,9 @@
   <data name="net_http_value_must_be_greater_than" xml:space="preserve">
     <value>The specified value must be greater than {0}.</value>
   </data>
+  <data name="ContentTypeInvalid" xml:space="preserve">
```

Comment:
Why do we need new entry here? Cannot we repurpose the one thrown for `textplain`?
If that one is not appropriate, then should we migrate it to this one as well?

Also, the error seems to be Mail specific - other entries start with `MailHeader`/`MailAddress` - if there is good reason to keep the new entry, shouldn't we unify the name?

Reply:
Ah, ok - it is just copy of `ContentTypeInvalid` into Common, that makes sense now.